.github/workflows/build-push.yml

```diff
@@ -10,7 +10,7 @@ env:
  CIBW_BEFORE_TEST: "pip install -r {project}/test-requirements.txt"
  CIBW_BEFORE_BUILD_LINUX: "rm -rf ~/.pyxbld && yum install -y libffi-devel libev libev-devel openssl openssl-devel"
  CIBW_ENVIRONMENT: "CASS_DRIVER_BUILD_CONCURRENCY=2 CFLAGS='-g0 -O3'"
- CIBW_SKIP: cp35* cp36* *musllinux* cp312*
+ CIBW_SKIP: cp35* cp36* *musllinux*
```

Comment:
and python 3.12 here?

Reply:
I've removed it from the skip. 

Before I was fixing the action, and upgrading cibuildwheel, without fixing python 3.12 fixed, it would have failed

cibuildwheel automatically builds all the version it know how

So after this merged we will have python 3.12 wheel